### ISSUE TITLE
Add image metadata support

### ISSUE THREAD

**Issue by skeggse** [garbados/eggchair#3]
Awesome project! Could be even cooler if you add support for info about the picture (bonus points for pulling metadata from the image itself)!

As part of that you could include user-submitted comments, though that might get ugly without authorizing the user through OpenID or the like.

**Comment by garbados** [comment]
What kind of metadata?

**Comment by skeggse** [comment]
It looks like this is just a collection of images with some kind of date or something to sort on, right? Maybe there are internal storage mechanisms, but I was thinking

`name`
`memo`
`dateTaken` (or something)

It's pretty minimal, but gives a little space for description.

**Comment by garbados** [comment]
Yep, it currently just uploads files with a timestamp to sort by. Future versions of eggchair will use [quilter](https://github.com/garbados/quilter), which also generates a file hash for quicker comparisons.

How would you associate the name and memo with each file?

**Comment by garbados** [comment]
I'd love to add an interface for adding stuff like tags and descriptions. Quilter should only compare creation date and attachment sizes, so that it doesn't needlessly update files back and forth whenever someone adds a tag.

Comments, ah... I'm sure I can make it happen, but each comment would be its own document, rather than nested in the image's JSON.
